plugins/modules/aci_l4l7_concrete_interface_attach.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l4l7_concrete_interface_attach
```

Comment:
do we need additional module for this or should we include into logic of M(ansible.aci.aci_l4l7_concrete_interface)?

Reply:
I'm not convinced that would work very well. It is possible to have multiple concrete interfaces attached to the same logical device on the APIC, so we would need the ability to add multiple attachments, and to delete existing attachments. I don't see how we would make that work from within M(cisco.aci.aci_l4l7_concrete_interface)